wallet/src/main/java/com/android/identity/issuance/IssuingAuthorityRepository.kt

```diff
@@ -34,4 +46,49 @@ class IssuingAuthorityRepository {
             it.configuration.identifier.equals(issuingAuthorityIdentifier)
         }
     }
+
+    private val observers = mutableListOf<Observer>()
+
+    /**
+     * Sets an observer to be notified when a credential has an updated state.
+     *
+     * Updates might be implemented using a lossy mechanism so applications must not
+     * rely on getting a callback whenever the state changes.
```

Comment:
It is not clear what it means in practice?

Reply:
I'm not sure what's not clear? I mean, our wallet server will be using push notifications to notify the app, these are lossy... any suggestion for improvement?

Reply:
Fair enough, updating this and its counterpart in `IssuingAuthority` to say this.